pkg/csi/plugins/nodeserver.go

```diff
@@ -313,14 +315,31 @@ func (ns *nodeServer) getRuntimeNamespacedName(volumeContext map[string]string,
 		runtimeName, nameFound := volumeContext[common.VolumeAttrName]
 		runtimeNamespace, nsFound := volumeContext[common.VolumeAttrNamespace]
 		if nameFound && nsFound {
+			glog.Infof("Get runtime namespace(%s) and name(%s) from volume context", runtimeNamespace, runtimeName)
```

Comment:
I think it should be set loglevel here.  Just like `glog.V(3)`

Reply:
Done